styles.css

```diff
@@ -0,0 +1,193 @@
+* {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  height: 30%;
+  width: 100%;
+  background-image: url("images/h.jpg");
+  color: white;
+  display: flex;
+  flex-direction: row;
+  align-content: space-between;
+  justify-content: space-between;
+}
+
+.logo,.cart {
+  padding: 5rem;
+}
+
+.cart {
+  font-weight: bold;
+}
+
+.total-cart {
+  display: flex;
+  flex-direction: row;
+  align-content: space-between;
+  justify-content: space-between;
+  font-weight: bold;
+}
+
+#cart-button {
+  width: 100%;
+}
+
+nav {
+  text-decoration: none;
+  color: white;
+
+}
+
+a {
+  text-decoration: none;
+  margin-right: 1rem;
+  margin-left: 1rem;
+  font-weight: bold;
+}
+
+a:visited {
+  text-decoration: none;
+  color: white;
+}
+
+.main-nav {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  padding-left: 15%;
+  width: 100%;
+  background-color: gray;
+  display: flex;
+  align-items: center;
+  margin-bottom: 2.5rem;
+
+}
+
+.container {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+
+.canvas {
+  width: 70%;
+  height: 100%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  align-content: center;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #00072b;
+  color: white;
+  height: 100%;
+  margin-top: 4rem;
+  padding: 5rem;
+}
+
+
+
+button {
+  width: 100%;
+  height: 1.7rem;
+  background-color: #3fb63f;
+  border: none;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  margin-top: .5rem;
+  color: white;
+  font-weight: bold;
+  padding: .5rem;
+}
+
+.highlight {
+  height: 20rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: space-between;
+  margin-bottom: 2.5rem;
+}
+
+.high-dog {
+  margin: 1rem;
+  width: 20rem;
+  display: inline-block;
+  margin-left: 0;
+}
+
+.high-desc {
+  margin: 1rem;
+  width: 50%;
+  margin-right: 0;
+  font-size: 130%
+}
+
+.dog-grid {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: space-between;
+  justify-content: space-between;
+}
+
+.dog-card {
+  width: 12rem;
+  height: 100%;
+  margin-bottom: 1rem;
+  font-size: 60%;
```

Comment:
I'm not a huge fan of percents for font size, but rather rems or ems. You can use decimals for rems, so you could use things like 0.6rem;